web/src/components/organisms/atoms/SideBar.jsx

```diff
@@ -0,0 +1,41 @@
+import Link from 'next/link';
+import { HiTemplate, HiOutlineShoppingBag, HiOutlineUsers } from "react-icons/hi";
+
+const Links = [
+    {
+      label: 'Dashboard',
+      icon: <HiTemplate />,
+      link: '/'
+    },
+    {
+      label: 'Orders',
+      icon: <HiOutlineShoppingBag />,
+      link: '/orders'
+    },
+    {
+      label: 'Staffs',
+      icon: <HiOutlineUsers />,
+      link: '/staffs'
+    }
+  ]
+
+const Sidebar = () => {
+    return (
+        <aside className="left-0 z-40 w-64 h-screen bg-gray-50 dark:bg-gray-800">
+            <div className="h-full px-3 overflow-y-auto">
```

Comment:
to much spaces on side, lets have padding left instead with value of 2
```suggestion
      <div className='h-full pl-2 overflow-y-auto'>
```